### PR TITLE
Remove salt and pepper (excess) words

### PR DESCRIPTION
**Pull request by gdevenyi**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -388,15 +388,15 @@ \section*{Introduction}
 both of which are easier if lessons have been designed by people with a shared world-view (Rule~4),
 and if lessons clearly signal what they cover (Rule~1).
 In particular,
-if lessons reference specific points in the model curriculum guidelines promulgated by many professional societies,
+if lessons reference specific points in the model curriculum guidelines promulgated by professional societies,
```
Comment (gvwilson) [inline paper.tex]: Hm - do all professional societies put out curriculum guidelines?

File: paper.tex [hunk 2]
```diff
@@ -388,15 +388,15 @@ \section*{Introduction}
...
 it can be much easier for people to find them and integrate them.
 
 Note also that individual lesson topics hinge on the learners having the proper prerequisite background,
 so this rule further emphasizes the importance of Rule~1.
 
 \rulemajor{Teach best practices for lesson development}
 
-Decades of pedagogical research has yielded many insights into
+Decades of pedagogical research has yielded insights into
 how best to build and deliver lessons \cite{hlw}.
```
Comment (gvwilson) [inline paper.tex]: Would prefer to retain "many" to emphasize how much is there.

**Review by gvwilson — COMMENTED** [review]
Minor comments.

**Comment by gdevenyi** [comment]
PR updated.

**Comment by gdevenyi** [comment]
Oh, stuff got merged into master in the meantime... rebasing again.

**Pull request merged**